hawkular-alerts-engine/src/test/resources/hawkular-alerts/alerts-data.json

```diff
@@ -0,0 +1,367 @@
+{
```

Comment:
well, I don't have a strong position about naming, I choosed alerts-data.json, as we are storing triggers + actions.
perhaps with alerts-data.json we can get rid of the hawkular-alerts/ folder in the future.
For this version I'm maintaining the "hawkular-alerts" folder.